net-misc/nextcloud-client/nextcloud-client-3.14.2.ebuild

```
@@ -0,0 +1,118 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Desktop Syncing Client for Nextcloud"
+HOMEPAGE="https://github.com/nextcloud/desktop"
+SRC_URI="
+	https://github.com/nextcloud/desktop/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz
+"
+S="${WORKDIR}/desktop-${PV/_/-}"
+
+LICENSE="CC-BY-3.0 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="doc dolphin nautilus test webengine"
+RESTRICT="!test? ( test )"
+
+# slot op for qtqui as this package uses private API parts of qtqui
+# src/gui/generalsettings.cpp:#include <private/qzipwriter_p.h>
+RDEPEND="
+	>=dev-db/sqlite-3.34:3
+	>=dev-libs/openssl-1.1.0:0=
+	dev-libs/glib:2
+	dev-libs/qtkeychain:=[qt6(+)]
```

In theory you are leaving the possibility of someone having pre-Qt6 enabled dev-libs/qtkeychain installed to attempt to upgrade to nextcloud-client-3.14.2, which is bound to fail. These things happen, so you can't just depend on unversioned qtkeychain with `qt6(+)` assumption.
```suggestion
	>=dev-libs/qtkeychain-0.14.2:=[qt6(+)]
```